pyuntl/untl_structure.py

```diff
@@ -1,8 +1,9 @@
 import socket
-import urllib2
-
+import json
+import urllib.request
+import urllib.error
+import urllib.parse
```

Comment:
We shouldn't be importing all three of these _every_ time we had `urllib2` in the original code, we should only be importing what we use. Same goes for other modules where this is done.

Reply:
Update the code with the suggested changes.